packages/@ember/routing/router.ts

```diff
@@ -1717,6 +1717,7 @@ function findRouteStateName(route: Route, state: string) {
   return routeHasBeenDefined(owner, router, stateName, stateNameFull) ? stateNameFull : '';
 }
 
+// TODO: rewrite `InternalOwner` to `Owner` by switching to `factoryFor`.
```

Comment:
When do you expect this to be addressed?

Reply:
In about ten minutes. :D Just keeping the types-only PR "clean".